src/DSharp.Compiler.Tests/Source/Member/Constants/Baseline.txt

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+define('test', ['ss'], function(ss) {
+  var $global = this;
+  // DSharp.Shell.src.UsageOfConstants
+
+  function UsageOfConstants() {
+  }
+  var UsageOfConstants$ = {
+    foo: function() {
+      var localIntConstant = 42;
+      var localFloatConstant = 42.42;
+      return 'Consts: ' + localIntConstant + '; ' + localFloatConstant + '; ' + '[example text constant]';
+    }
+  };
+
+
+  var $exports = ss.module('test', null,
+    {
+      UsageOfConstants: ss.defineClass(UsageOfConstants, UsageOfConstants$, [], null)
+    });
+
+  UsageOfConstants._copY_OF_TEXT_CONSTANT = '[example text constant]';
```

Comment:
Would we want this to generate still? I reckon we can remove the generation here too.

Reply:
I didn't touch the local constants. You are right this line is generated quite pointlessly because all usages should be translated to copy of literal, as well.